### DISCUSSION TITLE
🚀 The CP custom menu should be MSM site specific

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3572] [Feature Requests]
### Suggest an idea for ExpressionEngine

So the custom menu in the cp applies across all sites, but it would be really nice to be able to set it to site specific.  There are a lot of cases I can see where what's appropriate on one msm site isn't for another.

**Comment by ciaran-framework-design** [comment]
Also it would be great to add multiple menus (by role) 

EG: We could have a menu for a Super admin user and another for an Editor user

**Comment by intoeetive** [comment]
Aiming to get this added with https://github.com/ExpressionEngine/ExpressionEngine/pull/3719

**Comment by ciaran-framework-design** [comment]
Great

 

Thanks for the update
